tiledb/ml/models/_base.py

```diff
@@ -217,6 +217,7 @@ def _load_tensorboard(self, model_array: tiledb.Array) -> None:
             with open(path, "wb") as f:
                 f.write(file_bytes)
 
-    def _use_legacy_schema(self, model_array: tiledb.Array) -> bool:
+    @staticmethod
+    def _use_legacy_schema(model_array: tiledb.Array) -> bool:
```

Comment:
Is there any reason why this became `static`? I think again we call it on the `self` object of the class

Reply:
It just does a utility task, i.e., just checking if the model we try to load follows the old or the new schema. It could definitely live in a _utility-only_ file, but that's an overkill since this will be completely removed when we stop supporting the old model schema.